client/src/featureform/enums.py

```
@@ -6,7 +6,7 @@
 #
 
 from dataclasses import dataclass
-from enum import Enum
+from enum import Enum, auto
```

Ruff (0.8.2)

`enum.auto` imported but unused

Remove unused import: `enum.auto`

(F401)